content/installation/installing-manager.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: bt_wiki
+title: Installing and Configuring Cloudify Manager
+category: Installation
+draft: false
+weight: 100
+---
+A Cloudify Manager is a compute host that runs the Cloudify Management service. For version 4.3 and higher, you can install a single RPM file that installs Cloudify Manager with all of its dependencies. The Cloudify Manager RPM file is self-contained and does not require an internet connection during installation.
+
+{{% gsNote title="Prerequisites" %}}
+
+Make sure that your environment meets the [prerequisites]({{< relref "installation/prerequisites.md" >}}) before you install Cloudify Manager.
+
+{{% /gsNote %}}
+
+When you install Cloudify Manager, you can specify the private IP address, public IP address and administrator password as command parameters. If you use _-v_ for the cfy_manager command, you can see installation logs located at: ```/var/log/cloudify/manager/cfy_manager.log```. You can also specify installation parameters in the [config.yaml file]({{< relref "installation/installing-manager.md#changing-the-manager-parameters" >}}).
+
+{{% gsNote title="Administrator Password" %}}
+
+If you do not specify an administrator password, the installation process generates a random password and shows it as output when the installation is complete. We recommend that you specify an administrator password according to your security policy. 
+
+{{% /gsNote %}}
+
+You can install the [Cloudify CLI]({{< relref "installation/installing-cli.md" >}}) on a separate host to manage your Cloudify Manager remotely.
+
+## Installing Cloudify Manager
+
+_To install Cloudify Manager:_
+
+1. Go to the download page of the [Cloudify website](http://cloudify.co/download/).
+1. Copy the link address for the Cloudify Manager rpm file.
+1. From the CLI of your target host, run: ```sudo yum install \<rpm file link address>```
+   For example: ```sudo yum install http://cloudify-release-eu.s3.amazonaws.com/cloudify/4.3.0/release/cloudify-manager-install-4.3ga.x86_64.rpm```
+1. To change default configuration settings, edit the ```/etc/cloudify/config.yaml``` file.
+1. To install Cloudify Manager, run: ```cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>```
```

Comment:
I would mention here that you don't _have_ to specify private/public IPs here - you can also edit them in config.yaml. Some users might like this option as it makes creating an install script easier.

Reply:
Changing to: 1. To install Cloudify Manager, run: ```cfy_manager install [--private-ip <PRIVATE_IP>] [--public-ip <PUBLIC_IP>] [--admin-password <password>]```

  * If you specify the private and public IP addresses and the administrator password in the config.yaml file, do not specify them in the command options.
  * If you do not specify an administrator password in the command options or the config.yaml file, the installation process generates a random password and shows it as output when the installation is complete.